features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java

```diff
@@ -120,7 +120,7 @@ public Response rejected(String situationId, String feedback) {
                     LOG.debug("Situation {} already rejected", situationId);
                     return Response.accepted(MessageFormat.format("Situation {0} already rejected", situationId)).build();
                 }
-                feedback = String.format("reject situation [%s] -- user feedback :[%s]", situationId, feedback);
+                feedback = String.format("reject situation [%s]", situationId, feedback);
```

Comment:
Although extra argument won't cause error, it still better to remove it.

Reply:
We'll probably use this feedback later, but for now the frontend doesn't use it